starters/qwik-fetch-tailwind/src/components/button/button.tsx

```diff
@@ -0,0 +1,16 @@
+import { component$, PropFunction } from '@builder.io/qwik';
+
+export const buttonClassNames = 'w-28 h-28 rounded-full border-none bg-white shadow-lg text-6xl cursor-pointer my-0 mx-4 focus:outline-0';
```

Comment:
Just for consistency, should we move this `buttonClassNames` to `button.classNames.ts`, so it will be consistent with `header` component.

Reply:
Done 👍